licenses/admin.py

```diff
@@ -65,9 +65,12 @@ def _set_confirmed(self, queryset, value: bool):
 
         return updated
 
-    def changeform_view(
+    def change_view(
             self, request, object_id, form_url="", extra_context=None):
         """Don't show the save buttons if LR is confirmed."""
+        # TODO potentielle Nebenläufigkeitsprobleme?
+        self.exclude = []
```

Comment:
If we use this attribute on `self`, we should at least initialise it next to ordering, actions etc. above. Is this used in the implementation of the base class?

Reply:
It is mostly used in the [get_form](https://github.com/django/django/blob/19e838daa8872ee29fbea0bc471c2a6443f26835/django/contrib/admin/options.py#L760) and [get_formset](https://github.com/django/django/blob/19e838daa8872ee29fbea0bc471c2a6443f26835/django/contrib/admin/options.py#L2341) method of the base class to exclude other fields.

Reply:
[ef8dfd0](https://github.com/gocept/ok_tools/pull/63/commits/ef8dfd00b879c39d5cc1cf01a73e8ee513f41913)